<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1720</width>
    <height>880</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(50,50,50);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_File_Name">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>1290</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color:rgb(150,150,150);
	border:2px solid rgb(55,55,55);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>FILE_NAME</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Audio_File_Duration">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>620</y>
      <width>1290</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal
{
	height:10px;
	width:1270px;
	background:rgb(0,255,0);
	border-radius:5px;
}
QSlider::handle:horizontal
{
	background:rgb(150,150,150);
	width:20px;
	height:20px;
	margin:-7px -7px;
	border-radius:10px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Audio_Volume">
    <property name="geometry">
     <rect>
      <x>519</x>
      <y>780</y>
      <width>1191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal
{
	height:10px;
	width:1170px;
	background:rgb(0,255,0);
	border-radius:5px;
}
QSlider::handle:horizontal
{
	background:rgb(150,150,150);
	width:20px;
	height:20px;
	margin:-7px -7px;
	border-radius:10px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Volume">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>780</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:none;
	border-radius:15px;
	background-color:rgb(150,150,150);
	color:rgb(52,51,64);
	padding:0px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Value_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>660</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color:rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Value_2">
    <property name="geometry">
     <rect>
      <x>1640</x>
      <y>660</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color:rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>690</y>
      <width>1251</width>
      <height>82</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_Seek_Back">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border:none;
	border-radius:40px;
	background-color:rgb(150,150,150);
	color:rgb(52,51,64);
	min-height:80px;
	min-width:80px;
	max-height:80px;
	max-width:80px;
	padding:0px;
}
QPushButton:Hover
{
	background-color:rgb(200,200,200);

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Stop">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border:none;
	border-radius:40px;
	background-color:rgb(150,150,150);
	color:rgb(52,51,64);
	min-height:80px;
	min-width:80px;
	max-height:80px;
	max-width:80px;
	padding:0px;
}
QPushButton:Hover
{
	background-color:rgb(200,200,200);

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Play">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border:none;
	border-radius:40px;
	background-color:rgb(150,150,150);
	color:rgb(52,51,64);
	min-height:80px;
	min-width:80px;
	max-height:80px;
	max-width:80px;
	padding:0px;
}
QPushButton:Hover
{
	background-color:rgb(200,200,200);

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Pause">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border:none;
	border-radius:40px;
	background-color:rgb(150,150,150);
	color:rgb(52,51,64);
	min-height:80px;
	min-width:80px;
	max-height:80px;
	max-width:80px;
	padding:0px;
}
QPushButton:Hover
{
	background-color:rgb(200,200,200);

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Seek_Forward">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border:none;
	border-radius:40px;
	background-color:rgb(150,150,150);
	color:rgb(52,51,64);
	min-height:80px;
	min-width:80px;
	max-height:80px;
	max-width:80px;
	padding:0px;
}
QPushButton:Hover
{
	background-color:rgb(200,200,200);

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>1290</width>
      <height>561</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
    color: rgb(150, 150, 150) !important;
    border: 2px solid rgb(55, 55, 55);
}

QTableWidget::item {
    border-bottom: 1px solid rgb(55, 55, 55);
    padding: 5px; /* Ajusta el valor de padding según tus necesidades */
}

QTableWidget::item:selected {
    background-color: rgb(100, 100, 100);
}

QTableWidget QTableCornerButton::section {
    background-color: rgb(55, 55, 55);
    color: rgb(150, 150, 150) !important;
    border: 1px solid rgb(40, 40, 40);
    padding: 5px; /* Ajusta el valor de padding según tus necesidades */
}

QHeaderView::section {
    background-color: rgb(55, 55, 55);
    color: rgb(150, 150, 150) !important;
    border: 1px solid rgb(40, 40, 40);
    padding: 5px; /* Ajusta el valor de padding según tus necesidades */
}

QTableWidget::item:first {
    background-color: rgb(155, 155, 155);
}
</string>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Canción</string>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Artista</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Álbum</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Género</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Up-votes</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Down-votes</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1720</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="actionOpen_Audio_File"/>
   </widget>
   <addaction name="menuOpen"/>
  </widget>
  <action name="actionOpen_Audio_File">
   <property name="text">
    <string>Open Audio File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
